utils/cbits/DebugCallback.c

```diff
@@ -9,3 +10,18 @@ debugCallback(VkDebugUtilsMessageSeverityFlagBitsEXT messageSeverity,
   fprintf(stderr, "Validation: %s\n", pCallbackData->pMessage);
   return VK_FALSE;
 }
+
+VKAPI_ATTR VkBool32 VKAPI_CALL
+debugCallbackFatal(VkDebugUtilsMessageSeverityFlagBitsEXT messageSeverity,
+                   VkDebugUtilsMessageTypeFlagsEXT messageType,
+                   const VkDebugUtilsMessengerCallbackDataEXT *pCallbackData,
+                   void *pUserData) {
+  int severityError = messageSeverity & VK_DEBUG_UTILS_MESSAGE_SEVERITY_ERROR_BIT_EXT;
+  int isError = severityError = VK_DEBUG_UTILS_MESSAGE_SEVERITY_ERROR_BIT_EXT;
```

Comment:
Shouldn't this be `severityError & VK_D_U_M_S_E_B_E`?

Reply:
I mean, "if the flag is still set after being ANDed with the message"...